components/Person.js

```diff
@@ -15,111 +15,92 @@ const Container = styled.div`
   }
 `;
 
+const AvatarContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  justify-self: center;
+  align-self: center;
+`;
+
+const LogoNetwork = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  margin-right: auto;
+  align-self: center;
+  margin-left: 20px;
+  margin-right: 20px;
+
+  @media (min-width: 768px) {
+    flex-direction: row;
+  }
+`;
+
+const Description = styled.p`
+  font-family: "Montserrat";
+  font-size: 15px;
+  width: 80%;
+  color: #ffffff;
+  letter-spacing: 3px;
+  justify-self: center;
+  align-self: center;
+
+  @media (min-width: 768px) {
+    font-size: 20px;
+    align-self: start;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+`;
+
+const AvatarImage = styled.img`
+  width: 100%;
+  height: 100%;
+  border-radius: 50%;
+
+  @media (min-width: 768px) {
+    max-width: 400px;
+  }
+`;
+
+const MemberName = styled.h2`
+  font-family: "Orbitron";
+  font-size: 14px;
```

Comment:
`font-size` should follow the `rem` unit so it stays consistent when it scales. Like this
```css
.html {
  font-size: 14px;

  @media (min-width: 768px) {
    font-size: 16px;
  }
}
```
Then in the component
```javascript
const MemberName = styled.h2`
  font-size: 1rem;  // In this case, 1rem is 14px on mobile and 16px on desktop
` 
```